zmslayout/scss/atoms/list.scss

```
@@ -104,7 +104,7 @@ ul.list--half {
             content:            "\f061";
             display:            inline-block;
             float:              left;
-            font-family:        "Font Awesome 5 Free", "FontAwesome";
+            font-family:        "Font Awesome 7 Free", "FontAwesome";
```

Stylelint (17.14.1)

[error] Expected no quotes around "FontAwesome" (font-family-name-quotes)

(font-family-name-quotes)